Week1/homework/app.js

```diff
@@ -1,11 +1,158 @@
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
-  ];
+  // 1.1
+
+  // const myFavoriteBooks = [
+  //   '1984',
+  //   'white_fang',
+  //   'iron_heel',
+  //   'animal_farm',
+  //   'les_miserables',
+  //   'war_and_peace',
+  //   'crime_and_punishment',
+  //   'the_great_gasby',
+  //   'the_red_and_the_black',
+  //   'the_lord_of_the_rings',
+  // ];
+
+  // 1.3
+
+  // const listBooks = () => {
+  //   const ul = document.createElement('ul');
+  //   ul.classList.add('books');
+  //   ul.setAttribute('id', 'books-id');
+
+  //   myFavoriteBooks.forEach(book => {
+  //     const li = document.createElement('li');
+  //     li.classList.add('book');
+  //     li.textContent = book;
+  //     ul.append(li);
+  //   });
+
+  //   document.body.prepend(ul);
+  // };
+
+  // listBooks();
+
+  // 1.4
+  // title, language and author.
+
+  const books = {
+    1984: {
+      title: '1984',
+      language: 'English',
+      author: 'George Orwell',
+    },
+    white_fang: {
+      title: 'white fang',
+      language: 'English',
+      author: 'Jack London',
+    },
+    iron_heel: {
+      title: 'iron heel',
+      language: 'English',
+      author: 'Jack London',
+    },
+    animal_farm: {
+      title: 'animal farm',
+      language: 'English',
+      author: 'George Orwell',
+    },
+    les_miserables: {
+      title: 'les miserables',
+      language: 'English',
+      author: 'Victor Hugo',
+    },
+    war_and_peace: {
+      title: 'war and peace',
+      language: 'English',
+      author: 'Leo Tolstoy',
+    },
+    crime_and_punishment: {
+      title: 'crime and punishment',
+      language: 'English',
+      author: 'Fyodor Dostoevsky',
+    },
+    the_great_gasby: {
+      title: 'the great gasby',
+      language: 'English',
+      author: 'Scott Fitzgerald',
+    },
+    the_red_and_the_black: {
+      title: 'the red and the black',
+      language: 'English',
+      author: 'Henri Beyle',
+    },
+    the_lord_of_the_rings: {
+      title: 'the lord of the rings',
+      language: 'English',
+      author: 'J. R. R. Tolkien',
+    },
+  };
+
+  // 1.7
+
+  const covers = {
+    crime_and_punishment: 'books_cover/crime_punish.jpg',
+    white_fang: 'books_cover/white_fang.jpg',
+    animal_farm: 'books_cover/animal_farm.jpg',
+    iron_heel: 'books_cover/iron_heel.jpg',
+    the_lord_of_the_rings: 'books_cover/lord_rings.jpg',
+    war_and_peace: 'books_cover/war_peace.jpg',
+    les_miserables: 'books_cover/les_miserables.jpg',
+    the_great_gasby: 'books_cover/the_great_gasby.jpg',
+    1984: 'books_cover/1984.jpg',
+    the_red_and_the_black: 'books_cover/red_and_black.jpg',
+  };
+
+  const listBooksInDetail = bookList => {
+    Object.entries(bookList).forEach(book => {
+      //
+      const bookID = book[0];
+      const bookDetails = book[1];
+      // create a div element for each book and write them to body
+      const div = document.createElement('div');
```

Comment:
Even though the code works just fine, in future be mindful of using semantic tags while creating HTML elements. For example, if we need to create a list, we should use `ul` and `li` tags instead of using `div` tags. Also, we should be using `h1, h2, ..., h6` tags for headings instead of resized `p` or `span`. Not only it gives a better understanding of the HTML but is also helpful for accessibility.

In this case, when I run your application, I see an unordered list with a list of items(an item being a container of book image on left and some details on the right side. However, when we inspect the HTML in browser, I see a list of `div` tags with two `ul` tags and the first one without any `li` item in it.

It's fine for JS1-Week1 but please take care of it in the future 🙂